test/spec/vis/layers-factory.spec.js

```diff
@@ -145,6 +150,50 @@ describe('vis/layers-factory', function () {
         });
       });
     });
+
+    describe('Is high resolution screen', function () {
+      beforeEach(function () {
+        spyOn(LayersFactory, 'isRetina').and.returnValue(true);
+      });
+
+      describe('urlTemplateRetina is not defined', function () {
+        it("should not convert to '" + urlTemplateRetina + "'", function () {
+          var layerModel = layersFactory.createLayer('tiled', {
+            urlTemplate: urlTemplate
+          });
+
+          expect(layerModel.get('urlTemplate')).toEqual(urlTemplate);
+        });
+      });
+
+      describe('urlTemplateRetina is defined', function () {
+        it("should convert to '" + urlTemplateRetina + "'", function () {
```

Comment:
Same here

`it('should use urlTemplateRetina when defined');`

what do you think?

Reply:
👍